charms/layers/jupyter-notebook/templates/upstart.conf

```diff
@@ -0,0 +1,11 @@
+#!upstart
+description "jupyter"
+
+start on startup
+stop on shutdown
+
+respawn
+
+script
+jupyter notebook --no-browser
```

Comment:
This will run jupyter notebook as root. That's a security issue, but I'd wait to fix this until you do the spark integration, because spark might require you to run Jupyter under a specific user.

PS: What does the "user" config option do exactly? If it doesn't do anything, just remove it.